### ISSUE TITLE
Request release 6.1.1

### ISSUE THREAD

**Issue by illicitonion** [bazelbuild/buildtools#1157]
I just pulled in 6.1.0 and found that I was seeing buggy behaviour because https://github.com/bazelbuild/buildtools/pull/1148 treats stdin BUILD files as not needing formatting. It looks like https://github.com/bazelbuild/buildtools/pull/1155 fixes it - would it be possible to get a release out which contains this fix?

Thanks!

**Comment by fmeum** [comment]
@vladmos Could you include https://github.com/bazelbuild/buildtools/pull/1150/ in the release? Bazel will soon start to emit fixups using the new `use_repo_*` commands and that fix would help getting that right.

Also, if possible, could you create a tag of the form `v6.2.0`? Without the `v` prefix Go doesn't consider the tag a valid semantic version, which means that new releases aren't picked up automatically by Go dependency tooling.

**Comment by vladmos** [comment]
Hi,

Sure, I'll merge #1150 and make a new release tomorrow. I'm currently in a process of importing previous changes internally (takes some time because some of them are incompatible), hopefully won't take much time.

**Comment by fmeum** [comment]
@vladmos Would that include the `use_repo` command? I'm asking because I generally wouldn't expect 6.0.1 to have new features as a patch release.

**Comment by vladmos** [comment]
@fmeum Yes, it'll be a release from HEAD. Buildtools' versions are not very semantic because they follow the current version of Bazel (which is now 6.1.x, so the release will be also 6.1.x regardless of changes included).

**Comment by vladmos** [comment]
Done: https://github.com/bazelbuild/buildtools/releases/tag/6.1.1
